plan/planbuilder.go

```diff
@@ -69,8 +76,9 @@ type planBuilder struct {
 	inUpdateStmt bool
 	// colMapper stores the column that must be pre-resolved.
 	colMapper map[*ast.ColumnNameExpr]int
-
-	optFlag uint64
+	// Collect the visit information for privilege check.
+	visitInfo []visitInfo
```

Comment:
I don't think so, `visitInfo` says nothing about privilege, why name it `privilegeInfo`.